scipio/src/sys/uring.rs

```diff
@@ -634,14 +630,11 @@ impl UringCommon for SleepableRing {
                 SourceType::LinkRings(LinkStatus::Freestanding) => {
                     panic!("Impossible to have an event firing like this");
                 }
-                SourceType::Timeout(id @ Some(_), _) => {
-                    *id = None;
-                    Some(())
-                }
-                // This is actually possible: when the request is cancelled
-                // the original source does complete, and the cancellation
-                // would have marked us as false. Just ignore it.
-                SourceType::Timeout(None, _) => None,
+                SourceType::Timeout(_) => Some(()),
```

Comment:
As we no longer have this case, as a cleanup, we should remove the comments

Reply:
Oups totally meant to remove this, sent #138